spec/integration/knife/chef_repo_path_spec.rb

```diff
@@ -79,7 +79,8 @@
         EOM
       end
 
-      it "knife list --local -Rfp --chef-repo-path chef_r~1 / grabs chef_repo2 stuff", :windows_only do
+      # "Skipping for BK... As Windows 2019 has 8dot3name disabled by default"
+      it "knife list --local -Rfp --chef-repo-path chef_r~1 / grabs chef_repo2 stuff", :windows_only && :skip_buildkite do
```

Comment:
`:windows_only && :skip_buildkite` looks a bit funky here.  i suspect ruby just evaluates this to `true` all the time?

Reply:
Update as per comment